src/UserService.Models.Db/DbUser.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace LT.DigitalOffice.UserService.Models.Db
 {
     public class DbUser
     {
+        public const string TableName = "Users";
+        
         [Key]
```

Comment:
We use Fluent API, no need to use attributes

Reply:
done